src/libs/convertToLTRForComposer/index.android.ts

```diff
@@ -1,8 +1,53 @@
+import CONST from '@src/CONST';
 import ConvertToLTRForComposer from './types';
 
+/**
+ * Android only - We need to determinate when composer is considered empty this going to be used when we don't want to convert the input composer to LTR
+ */
+function hasBeenComposerConsideredEmpty(text: string): boolean {
+    // Regular expressions with all character that are consider as empty (spaces, unicode with spaces and unicode with at)
```

Comment:
Please update the comment to explain the empty expressions better. It might even be better to use three different variables so that it's more clear what the regexes are for. I'm looking for comments like this:

```
// The composer is considered "empty" if all it contains is the LTR character followed by an @. This handles
// the case where someone has RTL enabled and they began typing an @mention for someone. Without this,
// it leads to behavior where ... blah blah
const ltrRegex = new RegExp(`^${CONST.UNICODE.LTR}@$`);
```

With comments of this nature, it would be nearly impossible for someone to misinterpret why that specific regex is being used.

Reply:
Great suggestion thanks!, I have created the different variables and I have added comments to explain each one, please confirm.